utils/download_utils.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
```

Comment:
Consider adding a module/file-level docstring to describe the purpose of this file.